src/sr22_course_simulator/guidance/spiral_descent.py

```
@@ -0,0 +1,330 @@
+        aircraft_model=aircraft_model,
+        termination=combined_termination,
+        config=simulation_config,
+    )
+    simulation = replace(
+        simulation,
+        mode="maneuver_guidance",
+        notes=simulation.notes
+        + (
+            "Minimum implementation stops at 2,000 ft AGL; the source contingency to hold altitude and continue to the prescribed heading is not yet simulated.",
+            "Recovery after rollout is encoded in ManeuverSpec but not propagated by this guidance run.",
+        ),
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)